library/core/src/internal_macros.rs

```diff
@@ -190,7 +190,7 @@ macro_rules! impl_fn_for_zst {
 ///
 /// # Example
 ///
-/// ```
+/// ```ignore(cannot-test-this-because-non-exported-macro)
```

Comment:
I think that we have no way to run this doctest because `cfg_if!` is not exported and the doctest cannot import `cfg_if!`. Is that right?